### PR TITLE
fix: [CDS-50939]: final overview page added for gitops flow

### PR DESCRIPTION
**Pull request by satyam-harness**
#### Screenshots

NA

#### PR Checklist

<details>
<summary>Please check if your PR fulfils the following requirements</summary>

- [ ] Tests for the changes have been added. Ideally, include a test that fails without this PR but passes with it.
- [ ] Docs have been [added/updated](https://harness.atlassian.net/jira/software/c/projects/DOC/boards/40).
</details>

<details>
<summary>Use the following comments to re-trigger PR Checks</summary>

- Jest: `retrigger jest`
- Prettier: `retrigger prettier`
- Type Check: `retrigger typecheck`
- ESLint: `retrigger eslint`
- Sonar: `retrigger sonar`
- Standards: `retrigger standards`
- Build: `retrigger build`
- Title Check: `retrigger titlecheck`
- Feature Name Check: `trigger featurenamecheck`
- Coverage: `retrigger coverage`
- Rebase: `trigger rebase`
- Cypress Rest: `retrigger cypress-rest`
- Cypress CD: `retrigger cypress-cd`
- Cypress Pipeline: `retrigger cypress-pipeline`
- Cypress CV: `retrigger cypress-cv`
- Fix Prettier: `fix prettier`
</details>

### CODE REVIEW

**Comment by satyam-harness** [comment]
retrigger jest

**Comment by satyam-harness** [comment]
retrigger jest

File: src/modules/75-cd/pages/get-started-with-cd/Deploy/Deploy.tsx [hunk 1]
```diff
@@ -136,6 +164,155 @@ export const Deploy = ({ onBack }: { onBack: () => void }) => {
...
+                      <FormInput.Text
+                        name="name"
+                        style={{ width: '400px' }}
+                        placeholder={getString('name')}
+                        label={getString('name')}
+                      />
+                      <Layout.Horizontal flex={{ justifyContent: 'space-between' }}>
+                        <Layout.Horizontal>
+                          <Text font="normal" className={css.infoText}>{`${getString(
+                            'cd.getStartedWithCD.hostelGitopsAgent'
```
Comment (ishantmehtaharness) [inline src/modules/75-cd/pages/get-started-with-cd/Deploy/Deploy.tsx:185]: `hostelGitopsAgent` <> `hostedGitopsAgent`

File: src/modules/75-cd/pages/get-started-with-cd/Deploy/Deploy.tsx [hunk 2]
```diff
@@ -136,6 +164,155 @@ export const Deploy = ({ onBack }: { onBack: () => void }) => {
...
+                          <Text
+                            color={Color.GREY_900}
+                            padding={{ left: 'small' }}
+                            font="normal"
+                            className={css.infoText}
+                          >
+                            {agentData?.name}
+                          </Text>
+                        </Layout.Horizontal>
+                        {agentData?.health?.harnessGitopsAgent?.status === 'HEALTHY' && successsFullConfiguration}
```
Comment (ishantmehtaharness) [inline src/modules/75-cd/pages/get-started-with-cd/Deploy/Deploy.tsx:196]: What if its not successful?

**Review by ishantmehtaharness — APPROVED** [review]
Looks Good

**Review by tan-nhu — APPROVED** [review]

**Pull request merged**